anap1003/assignment1/Q1.java

```diff
@@ -0,0 +1,82 @@
+/*

```

Comment:
You can freely delete this type of comment and also @author comments. They are not useful, especially when code is created my multiple authors.